jerry-core/jerry.c

```diff
@@ -591,6 +591,41 @@ jerry_get_array_index_value (jerry_object_t *array_obj_p, /**< array object */
 } /* jerry_get_array_index_value */
 
 /**
+ * Get length of an array object
+ *
+ * Note:
+ *      Returns 0, if the given parameter is not an array object or
+ *      cannot retrieve the length.
```

Comment:
If it is an array you can always retreive the length. It is an unconfigurable data property.